test/python/algorithms/test_skip_qobj_validation.py

```diff
@@ -109,7 +109,15 @@ def test_w_noise(self):
             from qiskit.providers.aer.noise import NoiseModel
             from qiskit import Aer
 
-            self.backend = Aer.get_backend("qasm_simulator")
+            msg = (
+                "The method ``qiskit.circuit.quantumcircuit.QuantumCircuit.i()`` is deprecated as "
+                + "of qiskit-terra 0.45.0. It will be removed no earlier than 3 months after the "
+                + "release date. Use QuantumCircuit.id as direct replacement."
+            )
+
+            with self.assertWarnsRegex(msg, DeprecationWarning):
+                self.backend = Aer.get_backend("qasm_simulator")
```

Comment:
We probably don't actually want to _assert_ this, because when Aer fixes itself, it'll break our test suite.  Instead, we possibly just want to allow this particular warning if and only if it's emitted by Aer.

Reply:
Whoops, that's true!